src/lib/EventListener/ViewTemplatesListener.php

```diff
@@ -64,7 +64,7 @@ public function setViewTemplates(PreContentViewEvent $event): void
     }
 
     /**
-     * @return string[]
+     * @return array<class-string, mixed>
```

Comment:
```suggestion
     * @return array<class-string, string>
```
this is template map, this always should be strings.

Reply:
https://github.com/ibexa/user/pull/118/commits/77d1474538012d1b2f980e6b78a14c8db7eb938e